### PR TITLE
Add margin 0 to reset browser default margins

### PR DESCRIPTION
**Pull request by ghost**
Contributes to [simorgh#2330](https://github.com/bbc/simorgh/issues/2330)

**Overall change:** Add `margin: 0` to reset browser default margins

**Code changes:**

- Add `margin: 0` to  styles 

---

- [x] I have assigned myself to this PR and the corresponding issues
- [ ] Tests added for new features
- [ ] Test engineer approval

### CODE REVIEW

File: packages/components/psammead-visually-hidden-text/CHANGELOG.md [hunk 1]
```diff
@@ -3,6 +3,7 @@
 <!-- prettier-ignore -->
 | Version | Description |
 | --------------------- |
+| 1.0.4 | [PR#???](https://github.com/bbc/psammead/pull/???) Add margin:0 to reset any default browser default margins |
```
Comment (andrew-nowak) [inline packages/components/psammead-visually-hidden-text/CHANGELOG.md]:
```suggestion
| 1.0.4 | [PR#938](https://github.com/bbc/psammead/pull/938) Add margin:0 to reset any default browser default margins |
```

File: packages/components/psammead-visually-hidden-text/CHANGELOG.md [hunk 2]
```diff
@@ -3,6 +3,7 @@
 <!-- prettier-ignore -->
 | Version | Description |
 | --------------------- |
```
Comment (andrew-nowak) [inline packages/components/psammead-visually-hidden-text/CHANGELOG.md]:
```suggestion
 | ------- | ----------- |
```

This line is causing the markdown to not render correctly, could you fix it while you're here?

**Review by andrew-nowak — APPROVED** [review]
Looks good, just a couple of changelog fixes

**Review by j-pendlebury — APPROVED** [review]

**Comment by david-boydell** [comment]
Hard to verify this one, have statically checked margin is 0px though, so this is ready for merge.

**Pull request merged**